src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -329,7 +330,7 @@ public void testWriteThenReadStreamIntoByteArray ( int fileSize, int readBufferS
         byte[] dataRead = IOUtils.readStreamIntoByteArray(new FileInputStream(tempFile), readBufferSize);
 
         Assert.assertEquals(dataRead.length, dataWritten.length);
-        Assert.assertTrue(Arrays.equals(dataRead, dataWritten));
+        Assert.assertEquals(dataWritten, dataRead);
```

Comment:
Again, actual and expected values might be swapped here.

Reply:
done